### ISSUE TITLE
[Machine learning] Building an image classifier in Python

### ISSUE THREAD

**Issue by 07munene** [section-engineering-education/engineering-education#5236]
#### Proposed title of article
Building an image classifier in Python

#### Proposed article introduction
An image classifier is a program that labels an image according to its visual content by focusing on the relationship of the nearby pixels.
In this article, we are going to learn how to use the Convolutional Neural Network(CNN) model to create the image classifier. The CNN model is created using TensorFlow.
An image classifier is used in areas such as medical imaging, object identification in satellite images, in self driven cars, guiding autonomous robots and many more.

#### Key takeaways 
1. What is image classification
2. How to build an image classifier
3. Structure of an image classification task
4.  Image classification techniques
5. Creating a Convolutional Neural Network

#### Article quality
The article is indepth in content and provides a step by step process on how to go about creating the image classifier and some of the expected setbacks one may experience.

#### References
N/A

**Comment by ahmadmardeni1** [comment]
Thank you for submitting your topic. @07munene 
After some careful consideration, it struck us that this topic may be a bit over-saturated throughout other blog sites and official documentation.

We believe this is the best way for students to build a great portfolio (for potential employers) is by building what does not exist and what can provide the most value.

Please feel free to suggest an alternate topic to explore. 🚀
